checks.d/network.py

```diff
@@ -440,3 +446,117 @@ def _parse_solaris_netstat(self, netstat_output):
             metrics_by_interface[iface] = metrics
 
         return metrics_by_interface
+
+    @staticmethod
+    def _linux_sockstat_get_conntrack_max():
+        proc_name = '/proc/sys/net/ipv4/ip_conntrack_max'
+        try:
+            proc = open(proc_name,'r')
+            try:
+                lines = proc.readline()
+            finally:
+                proc.close()
+            return int(lines)
+        except IOError:
+            return 0
+
+    @staticmethod
+    def _linux_sockstat_get_conntrack_count():
+        proc_name = '/proc/sys/net/ipv4/netfilter/ip_conntrack_count'
+        try:
+            proc = open(proc_name,'r')
+            try:
+                lines = proc.readline()
+            finally:
+                proc.close()
+            return int(lines)
+        except IOError:
+            return 0
```

Comment:
I'm not entirely sure we want to return report a `0` value when we encounter an `IOError`. This seems to have the potential to misguide